### ISSUE TITLE
Incorrect result

### ISSUE THREAD

**Issue by youdly** [adamcaudill/hstspreload.com#5]
`google.com` and `facebook.com` return `null` for Firefox and Tor.

```
$ curl https://hstspreload.com/api/v1/status/google.com
{"domain":"google.com","chrome":{"present":true,"include_subdomains":true,"last_updated":1512439204},"firefox":null,"tor":null}
```
```

$ curl https://hstspreload.com/api/v1/status/facebook.com
{"domain":"facebook.com","chrome":{"present":true,"include_subdomains":false,"last_updated":1512439205},"firefox":null,"tor":null}
```

`nextcloud.com`, `skype.com` and [pinimg.com](https://github.com/adamcaudill/hstspreload.com/issues/4) return `null` in all three lists.

Checked with Firefox [list](https://dxr.mozilla.org/mozilla-central/source/security/manager/ssl/nsSTSPreloadList.inc).

btw, there is official github [mirror](https://github.com/mozilla/gecko-dev) of firefox. it might be better to parse the [raw file](https://github.com/mozilla/gecko-dev/raw/master/security/manager/ssl/nsSTSPreloadList.inc), instead of html.

**Comment by adamcaudill** [comment]
I know this response is absurdly late - I've checked all those mentioned, and the data is currently correct. I'm not sure if this was fixed while making other changes, of if there was a processing dely / caching issue at the time, but at this point I've compared the data in all three lists to what the API is returning, and it seems to be correct.

Sorry again for it taking so long for me to notice this issue and check the data.
